tensorflow/core/ops/nn_ops.cc

```diff
@@ -632,9 +632,7 @@ REGISTER_OP("_FusedDepthwiseConv2dNative")
     // Attributes for the LeakyRelu ----------------------------------------- //
     .Attr("leakyrelu_alpha: float = 0.2")
     // ---------------------------------------------------------------------- //
-
     .SetShapeFn(shape_inference::DepthwiseConv2DNativeShape);
-
 // --------------------------------------------------------------------------
```

Comment:
Nit: Please add newlines around this line. (One above, and one below.) (It's a section separator for DepthwiseConv2D ops.)